src/style.css

```diff
@@ -0,0 +1,24 @@
+html {
+  margin: 0;
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 13px;
+}
+
+.heading {
+  font-size: 17px;
+  font-weight: bold;
```

Comment:
font-weight: bold is a default setting for h1, remove

Reply:
corrected all comments